src/components/Asset/StakingHistoryTab.tsx

```diff
@@ -158,23 +158,22 @@ const FPRHistory: React.FC<FPRHistoryProps> = ({ fpr, asset }) => {
             </HistoryWrapper>
           </FPRRow>
         ) : (
-          <FPRRow isStakingRoyalties={false}>
+          <FPRRow isStakingRoyalties={false} span={2}>
```

Comment:
also span 2 if it has data to render.
![image](https://github.com/klever-io/kleverscan/assets/39703077/65e876b2-0f34-4c39-b360-ce6bde79512d)